pkg/cloud/openstack/services/loadbalancer/loadbalancer.go

```diff
@@ -187,7 +191,6 @@ func (s *Service) ReconcileLoadBalancerMember(clusterName string, machine *clust
 	}
 
 	loadBalancerName := fmt.Sprintf("%s-cluster-%s-%s", networkPrefix, clusterName, kubeapiLBSuffix)
```

Comment:
Can you add this log message again? Seems like a rebase mistake to me (concurrent change on master & file move)

Reply:
I added the log message again due to accidentally removed.

Reply:
Still removed in the diff :)

Reply:
Now really added 👍